include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -5445,7 +5457,8 @@ inline DEAL_II_ALWAYS_INLINE Tensor<1, n_components_, VectorizedArrayType>
          internal::ExcAccessToUninitializedField());
 #  endif
 
-  Assert(this->normal_x_jacobian != nullptr, ExcNotInitialized());
+  Assert(this->normal_x_jacobian != nullptr,
+         internal::MatrixFreeNotInitialized("???"));
```

Comment:
`update_gradients` In principle one should also check for the normal vectors, but we force this to be done at the moment here: https://github.com/dealii/dealii/blob/5163ed818e5a008bdcce7ea53317dca7c6e250a0/include/deal.II/matrix_free/mapping_info.templates.h#L325-L330